x-pack/plugins/transform/kibana.json

```diff
@@ -19,7 +19,8 @@
     "security",
     "usageCollection",
     "spaces",
-    "alerting"
+    "alerting",
+    "savedObjectsTagging"
```

Comment:
@davismcphee since tags don't show up in transforms, and saved_objects_finder is not used, we could revert those changes inclusive translations, right?

Reply:
Removed these changes